lib/grim/null_logger.rb

```diff
@@ -0,0 +1,11 @@
+require "logger"
+
+module Grim
+  class NullLogger < ::Logger
```

Comment:
Personally, I wouldn't inherit from logger. I'd just have null logger and define the debug method (and info if that is used anywhere).